package com.java.javapractice.ch08_ExceptionHandling;

class ExceptionEx11 {
    public static void main(String[] args) {
        throw new RuntimeException(); //고의로 RuntimeException을 발생시킨다.
    }
}
